backend/core/src/main/java/com/feed_grabber/core/config/NotificationService.java

```diff
@@ -15,6 +15,7 @@ public NotificationService(SimpMessagingTemplate simpMessagingTemplate) {
     }
 
     public <T> void sendMessageToConcreteUser(String userId, String topic, T payload) {
+        System.out.println(userId);
```

Comment:
Maybe you should delete this println statement?

Reply:
Yeap, thnx